src/app/modules/visualizacion-informacion/components/bi-queries/bi-queries.component.ts

```diff
@@ -52,6 +55,7 @@ export class BiQueriesComponent implements OnInit {
         if (this.consultaSeleccionada) {
             this.argumentos = this.consultaSeleccionada.argumentos;
             this.organizaciones$ = this.auth.organizaciones();
+            this.zonasSanitarias$ = this.zonaSanitariaService.search();
```

Comment:
this.zonaSanitaria$ = this.zonaSanitariaService.search().pipe(
      cache()
    );

Reply:
leesto

Reply:
Ese `cache` sirve para cuando hay más de una subscripción al observable (evita que se ejecute dos veces el GET). En este caso hay una sola (`async`).

Lo ideal sería en el servicio agregar 
```
   cacheStorage({ key: 'zonas-sanitarias' , ttl: 60 * 24 })
```

Así se cachea en toda la aplicación. 

Además no es necesario asignarlo todo el tiempo, en el inicio de la componente basta y sobra.